core/config.h

```diff
@@ -384,7 +384,7 @@
 #define RESOURCE_REGISTRATION_NUM_MAX 16
 
 /* Max length of resource/event url */
-#define RESOUCE_EVENT_URL_LEN_MAX 256
+#define RESOURCE_EVENT_URL_LEN_MAX 256
```

Comment:
Maybe we can keep it unchanged now, and define and use the macro with correct name in wamr-app-framework, and after that, remove this macro in WAMR, so that it won't lead to compilation error in wamr-app-framework.

Reply:
> I can't find any other usage in the repository, and even an internet search doesn't turn up any other results, but I will revert this anyway.

https://github.com/bytecodealliance/wamr-app-framework/blob/8752a059be5a796db31ee6dc48e4e915f5017424/app-mgr/app-manager/resource_reg.c#L161